src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = "1,2")
+    void splitTest(String input) {
```

Comment:
```suggestion
    @CsvSource(value = {"1,2:,"}, delimiter = ':')
    void splitTest(String input, String separator) {
```
ValueSource 만 사용을 하셨는데 이유가 궁금합니다!!
CsvSource 를 활용해서 메소드 파라미터에 String input, String separator 구분을 시켜줘도 좋을 것 같아요

Reply:
@CsvSource 가 좀 익숙하지 않아서 @ValueSource 만 썼었어요.
말씀하신대로 @CsvSource 도 적극적으로 써봐야겠네요!